android/gradle.properties

```diff
@@ -21,3 +21,8 @@ android.useDeprecatedNdk=true
 
 # Disable incremental resource processing as it broke relase build
 android.enableAapt2=false
+
+MYAPP_RELEASE_STORE_FILE=my-release-key.keystore
+MYAPP_RELEASE_STORE_PASSWORD=blahblahblah
+MYAPP_RELEASE_KEY_PASSWORD=blahblahblah
+MYAPP_RELEASE_KEY_ALIAS=my-key-alias
```

Comment:
Okay for now, but let's definitely not let this get committed and merged :)

Reply:
haha! yes. it was just the easiest way to get it building completely :P. i plan to tidy up a lot before the 'proper' pull request.